app/controllers/branding_groups_controller.rb

```diff
@@ -90,6 +101,16 @@ def destroy
     end
   end
 
+  # helper to merge in the list of curator IDs into the :user_ids parameter
+  # will prevent curator from removing themselves from the collection
+  def self.merge_curator_params(curator_list, collection, user)
+    curators = curator_list.split(',').map(&:strip)
+    user_ids = curators.map { |email| User.find_by(email: email)&.id }.compact
+    # ensure current user cannot remove accidentally remove themselves from the list
+    user_ids << user.id if collection.present? && collection.users.include?(user) && !user_ids.include?(user.id)
+    user_ids
```

Comment:
Let's refactor this method to return and operate on `users` rather than user_ids.  Unless you're making high-performance plucks (and you know I loooove my plucks), it's generally better and faster to just operate on the objects themselves

Reply:
OK I'll try to get that working

Reply:
e6441de